spec.bs

```diff
@@ -2859,6 +2864,175 @@ This specification defines two [=policy-controlled features=] identified by the
 Issue(WICG/turtledove#522): Move from "`*`" to "`self`".
 
 
+# Handling Direct from Seller Signals # {#handling-direct-from-seller-signals}
+
+This section specifies a manner by which signals may be provided to auctions such that the signals
+are only used within their intended auction.
+
+Scripts running on the page may peform a {{WindowOrWorkerGlobalScope/fetch()}} call using the
+`{adAuctionHeaders: true}` option, which causes the value of the [:Ad-Auction-Signals:] response
+header on that request to be used to provide signals in auctions running on the page whose
+seller matches the origin of the {{WindowOrWorkerGlobalScope/fetch()}} destination.
+
+<div algorithm="fetch capture adAuctionHeaders boolean patch">
+Modify the definition of a [=request=]: 
+
+A [=request=] has an associated boolean <dfn for=request>capture-ad-auction-headers</dfn>.
+Unless stated otherwise it is false.
+</div>
+
+<div algorithm="fetch capture adAuctionHeaders RequestInit patch">
+Modify [[FETCH]]'s {{RequestInit}} dictionary to add an adAuctionHeaders attribute:
+
+<pre class=idl>
+partial dictionary RequestInit {
+  boolean adAuctionHeaders;
+};
+</pre>
+</div>
+
+<div algorithm="fetch new request patch">
+The following step will be added to the <code><a constructor for="Request" lt="Request()">new Request (<var ignore>input</var>, <var ignore>init</var>)</a></code>
+constructor steps, before step "Set this's [=Request/request=] to |request|":
+
+1. If <var ignore=''>init</var>["{{RequestInit/adAuctionHeaders}}"] <a for=map>exists</a>, then set |request|'s

```

Comment:
For consistency, I'd write `[=map/exists=]`

Reply:
Done